### PR TITLE
Use 'find -L . -type -f | wc -l' rather than 'ls | wc -l' to count files

### PR DESCRIPTION
**Pull request by mrquincle**
Fixes https://github.com/jarv/cmdchallenge/issues/21

### CODE REVIEW

File: challenges.yaml [hunk 1]
```diff
@@ -192,12 +192,12 @@
       Count the number of files in the current
```
Comment (jarv) [inline challenges.yaml]: since this changes the correct answer, can you bump the version to 6?

File: challenges.yaml [hunk 2]
```diff
@@ -192,12 +192,12 @@
...
       working directory. Print the number of
       files as a single integer.
-    example: ls | wc -l
-    example_fail: echo 2
```
Comment (jarv) [inline challenges.yaml]: would prefer to make this `example_fail: echo 3`

Comment (mrquincle) [inline challenges.yaml]: Your choice, but isn't that obvious? :-)

File: challenges.yaml (hunk 1 above)
Comment (mrquincle) [inline challenges.yaml]: Sure

File: challenges.yaml (hunk 2 above)
Comment (mrquincle) [inline challenges.yaml]: I updated it and ran it locally. Why does the build fail? You just use the output in `expected_output` isn't it?

**Comment by jarv** [comment]
Recent change added randomizer scripts that check the submitted command against random value. If you update this file https://github.com/jarv/cmdchallenge/blob/master/ro_volume/randomizers/count_files#L5 with your new example solution `find -L . -type f | wc -l` it will pass.

File: challenges.yaml (hunk 2 above)
Comment (jarv) [inline challenges.yaml]: The reason I put the `example_fail` in the yaml file is so the test harness can test commands that have the same output as the correct solution, but are not correct. Needs better docs (this was a recent change).

**Comment by mrquincle** [comment]
Handled all your remarks.

**Pull request merged**